plugin/src/main/cpp/include/export/android_export_plugin.h

```diff
@@ -0,0 +1,87 @@
+/**************************************************************************/
+/*  android_export_plugin.h                                               */
+/**************************************************************************/
+/*                       This file is part of:                            */
+/*                              GODOT XR                                  */
+/*                      https://godotengine.org                           */
+/**************************************************************************/
+/* Copyright (c) 2022-present Godot XR contributors (see CONTRIBUTORS.md) */
+/*                                                                        */
+/* Permission is hereby granted, free of charge, to any person obtaining  */
+/* a copy of this software and associated documentation files (the        */
+/* "Software"), to deal in the Software without restriction, including    */
+/* without limitation the rights to use, copy, modify, merge, publish,    */
+/* distribute, sublicense, and/or sell copies of the Software, and to     */
+/* permit persons to whom the Software is furnished to do so, subject to  */
+/* the following conditions:                                              */
+/*                                                                        */
+/* The above copyright notice and this permission notice shall be         */
+/* included in all copies or substantial portions of the Software.        */
+/*                                                                        */
+/* THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,        */
+/* EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF     */
+/* MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. */
+/* IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY   */
+/* CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,   */
+/* TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE      */
+/* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                 */
+/**************************************************************************/
+
+#pragma once
+
+#include <godot_cpp/classes/editor_plugin.hpp>
+
+#include "export_plugin.h"
+
+using namespace godot;
+
+namespace android {
+static const int EYE_TRACKING_NONE_VALUE = 0;
+static const int EYE_TRACKING_OPTIONAL_VALUE = 1;
+static const int EYE_TRACKING_REQUIRED_VALUE = 2;
+
+static const int HAND_TRACKING_NONE_VALUE = 0;
+static const int HAND_TRACKING_OPTIONAL_VALUE = 1;
+static const int HAND_TRACKING_REQUIRED_VALUE = 2;
+} // namespace android
```

Comment:
Again, maybe these should be class constants rather than in a namespace?

Reply:
The namespace was added to avoid constants collision between the various vendors; they have similar features but different set of values so we can't have a single set of constants for them all.

Reply:
Yeah, I just meant that the constants could be members of the two classes rather than two namespaces.

So, these could go inside `AndroidExportPlugin` as like `AndroidExportPlugin::EYE_TRACKING_NONE_VALUE`, rather than the namespace `android::EYE_TRACKING_NONE_VALUE`.

Reply:
Good point! Let me make the update.